packages/sit-onyx/src/components/OnyxMiniSearch/OnyxMiniSearch.vue

```diff
@@ -45,13 +47,18 @@ defineExpose({
 </script>
 
 <template>
-  <div :class="['onyx-mini-search', densityClass]" v-bind="rootAttrs">
+  <div
+    :class="['onyx-mini-search', densityClass]"
+    v-bind="rootAttrs"
+    :style="{ '--onyx-placeholder-character-count': placeholder?.length }"
```

Comment:
```suggestion
    :style="{ '--onyx-placeholder-character-count': placeholder.length }"
```

The placeholder is always defined :)